docs/deployment/infrastructure.md

```diff
@@ -8,18 +8,18 @@ The infrastructure is configured as code via [Terraform](https://www.terraform.i
 
 ```mermaid
 flowchart LR
-    %% DMV integration is currently disabled, hence the lines commented out below
-
-    %% dmv[DMV Eligibility Verification API]
     benefits[Benefits application]
     style benefits stroke-width:5px
     recaptcha[Google reCAPTCHA]
     rider((User's browser))
     idg[Identity Gateway]
-    mst_elig[Eligibility Server]
-    cc_data[(Courtesy Card data)]
+    elig_server[Eligibility Server]
+    cc_data[(Agency Card data)]
```

Comment:
Minor suggestion: rename `cc_data` to `ac_data` to prevent any future confusion

Reply:
Fixed in [`bceb52a`](https://github.com/cal-itp/benefits/pull/1361/commits/bceb52ad1e6adde22099661e1971e77763d6d2ae)